openml/tasks/functions.py

```diff
@@ -235,7 +236,7 @@ def get_task(task_id):
 
     # TODO look into either adding the class labels to task xml, or other
     # way of reading it.
-    class_labels = dataset._retrieve_class_labels()
+    class_labels = dataset._retrieve_class_labels(task.target_feature)
```

Comment:
why is this called ``target_feature``? That's not a feature, is it? ``target_value``? ``target_name``? ``target_column``?